### PR TITLE
Bcl stage updates

### PR DESCRIPTION
**Pull request by DavidGoldwasser**
@nllong and @kflemin There are no functional changes here, just updates to file names to support upload to BCL. I ran measure tests for all measures that changed using OpenStudio CLI before commit and upload to BCL. A few measures still had to be pushed to BCL using web interface instead of rake task.

Here is one more thing we may want to fix before this is merged. I thought you might both have ideas.

When I run the `openstudio measure -t` two measures always update even if I run it multiple times without any changes. Seems the version modified changes which changes the version ID

Screen Shot 2020-06-23 at 2 51 54 PM

When I run `rake openstudio:update_measures` in addition two the two above, 5 additional measures change with new blank line in the `report.html.erb` file. These are all reporting measures. If I run it 6 times it will keep adding an extra line each time.

Screen Shot 2020-06-23 at 2 53 42 PM

I'm not sure how to prevent either of these, any thoughts? measure.xml files have always been a bit sensitive, but I thought we had addressed most of these.

### CODE REVIEW

**Comment by DavidGoldwasser** [comment]
Just a as a heads up 3 measures in model articulation gem also update version modified, so if we find fix here can do there as well. (DOE Prototype, DEER prototype, create baseline)

**Comment by kflemin** [comment]
confirming the blank lines in common-measures and the  3 same measures updating version_modified in model_articulation.  The rake openstudio:update_measures calls openstudio measure -t as well, but passes in a bundle and bundle_path: 

/Applications/OpenStudio-3.0.0/bin/openstudio --verbose --bundle '/Users/kflemin/repos/openstudio-model-articulation-gem/Gemfile' --bundle_path '/Users/kflemin/repos/openstudio-model-articulation-gem/.bundle/install/' measure -t '/Users/kflemin/repos/openstudio-model-articulation-gem/lib/measures'

File: lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
 
 
+
```
Comment (nllong) [inline lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb:40]: are these the random additions that are happening?

File: lib/measures/openstudio_results/resources/os_lib_schedules.rb [hunk 2]
```diff
@@ -547,31 +547,33 @@ def self.createComplexSchedule(model, options = {})
...
-      day_schedule = rule.daySchedule
-      day_schedule.setName("#{sch_ruleset.name} #{data_array[0]}")
-      data_array.delete_at(0)
-      data_array.delete_at(0)
-      data_array.delete_at(0)
-      data_array.each do |data_pair|
-        hour = data_pair[0].truncate
-        min = ((data_pair[0] - hour) * 60).to_i
-        day_schedule.addValue(OpenStudio::Time.new(0, hour, min, 0), data_pair[1])
+    unless options['rules'].nil?
```
Comment (nllong) [inline lib/measures/openstudio_results/resources/os_lib_schedules.rb:550]: The `&.` is the recommended way of handling this now. Is that the only change in this block?

**Comment by nllong** [comment]
good to merge this @DavidGoldwasser

**Comment by DavidGoldwasser** [comment]
@nllong yes, good to merge.

File: lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb (hunk 1 above)
Comment (DavidGoldwasser) [reply lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb:40]: Well, not random happens for a few measures every time update measures rake task is called, but  `openstudio measure -u` in the CLI.

Comment (DavidGoldwasser) [reply lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb:40]: Can figure out why it happens here and bypass it up resolve it in extension gem. Then can go back and clean up blank lines in the measures.

**Pull request merged**